examples/lit-icd-app/lit-icd-common/lit-icd-server-app.matter

```diff
@@ -1670,6 +1670,7 @@ endpoint 0 {
     ram      attribute clusterRevision default = 1;
 
     handle command OpenCommissioningWindow;
+    handle command OpenBasicCommissioningWindow;
```

Comment:
Why is this being enabled?  Where is it used?  And why?

Reply:
Just to make the device can be commissioned to the second fabric easily. If using the two controller (devCtrl and devCtrl2) in one same container/host to execute this test, the accessory can still resolve the address for first controller even if the first controller is shutdown by 'self.devCtrl.Shutdown()'. So I used two containers for two fabrics. If using OpenBasicCommissioningWindow from the first controller, the second could use the original setup pincode to commission the server device.

Reply:
thanks for clarification

Reply:
Would follow-up it if it is still concerning. thanks

Reply:
So is the issue here that you can't communicate the right setup code from OpenCommissioningWindow to the other container?

The problem here is that this example was explicitly not enabling OpenCommissioningWindow because it's meant to be an example of how you actually do a real app.  We are not messing that up because of our testing limitations.  We should really not be doing that @wqx6 @yunhanw-google